test/cypress/integration/payment-links/delete_payment_links_spec.js

```diff
@@ -0,0 +1,52 @@
+const commonStubs = require('../../utils/common_stubs')
+const { getProductsStub, getProductByExternalIdStub, deleteProductStub } = require('../../utils/products_stubs')
+const userExternalId = 'a-user-id'
+const gatewayAccountId = 42
+const productExternalId = 'a-product-id'
+
+const buildPaymentLinkOpts = function buildPaymentLinkOpts (name, href, language) {
+  return {
+    external_id: productExternalId,
+    name: name,
+    language: language,
+    type: 'ADHOC',
+    links: [
+      {
+        'rel': 'friendly',
+        'method': 'GET',
+        'href': href
+      }
+    ]
+  }
+}
+
+const product = buildPaymentLinkOpts('Pay for a parking permit', 'example.com/my-service/pay-for-a-parking-permit', 'en')
```

Comment:
As we only call this function once, I don't think its worthwhile having a function at all, just make it a constant. I.e.
```
const product = { external_id: productExternalId }
```